cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -159,7 +160,11 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
     ).convert { name ->
         allPackageManagersByName[name]
             ?: throw BadParameterValue("Package managers must be one or more of ${allPackageManagersByName.keys}.")
-    }.split(",")
+    }.split(",").deprecated(
+        message = "--package-managers is deprecated, use -P ort.analyzer.enabledPackageManagers on the ort command " +
+                "instead.",
+        tagValue = "use -P ort.analyzer.enabledPackageManagers=[] on the ort command instead"
```

Comment:
This still has `=[]`. Also, I wonder whether instead of dropping it completely, we should write something like `=...` (I haven't checked how we might already be doing this in other places).